draft-ftbs-rats-msg-wrap.md

```diff
@@ -149,6 +156,15 @@ number ({{Section 12.3 of -coap}}).
 : The RATS conceptual message serialized according to the
 value defined in the type member.
 
+`ind`:
+: An optional bitmap that indicates which conceptual message(s) types
+are carried in the `value` field.  Any combination (i.e., any value
+between 1 and 15, included) is allowed.  This is useful only if the
+`type` is potentially ambiguous, for example if the base media type is
+not profiled (e.g., `application/eat+cwt`) or if the same profile
```

Comment:
```suggestion
not profiled (e.g., `application/eat+cwt`) or if the `value` field contains multiple values having different `cm-type` values or if the same profile
```

Reply:
done in https://github.com/thomas-fossati/draft-ftbs-rats-msg-wrap/pull/29